src/components/_internal/BaseDateRangePicker/BaseDateRangePicker.tsx

```diff
@@ -1,15 +1,9 @@
-import React from 'react';
-import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { any, prop } from 'ramda';
 import { isNotNull } from 'ramda-adjunct';
 
 import { dateRangePickerStyles } from './styles';
-import {
-  BaseDateRangePickerPropTypes,
```

Comment:
You should also remove the custom proptypes definitions that are imported into file (unless it is used also elsewhere). We don't want to end up with unused code.

Reply:
Good catch. Fixed!

Reply:
I'm gonna fix the errors.